src/explorer/Main.hs

```diff
@@ -43,19 +46,28 @@ import           Pos.Util.UserSecret        (usVss)
 import           Pos.Util.Util              (powerLift)
 import           Pos.WorkMode               (RealMode, WorkMode)
 
+import           Pos.Explorer               (ExplorerBListener,
+                                             runExplorerBListener)
 import           Pos.Explorer.Socket        (NotifierSettings (..))
 import           Pos.Explorer.Web           (explorerPlugin, notifierPlugin)
 
 import           ExplorerOptions            (Args (..), getExplorerOptions)
 import           Params                     (getBaseParams, getKademliaParams,
                                              getNodeParams, gtSscParams)
 
+
+type ExplorerProd = ExplorerBListener (RealMode SscGodTossing)
+
+-- | Lift monadic computation to 'ExplorerBListener'.
+liftBListenerRedirects :: m a -> ExplorerBListener m a
+liftBListenerRedirects = coerce
```

Comment:
I am not sure what `coerce` does here. I hope it is good. Looks like `coerce` is synonim for `lift` ? (because `coerce` sounds much scarier then `lift`)